prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,10 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+const firstElement = languages;
+console.log(firstElement[0]);
+const thridElement = 'javascript';
```

Comment:
you need to access the value using the languages array and the proper index value